controllers/metallb_controller.go

```diff
@@ -67,7 +70,13 @@ var ManifestPath = "./bindata"
 
 func (r *MetallbReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	_ = context.Background()
-	_ = r.Log.WithValues("metallb", req.NamespacedName)
+	logger := r.Log.WithValues("metallb", req.NamespacedName)
+
+	if req.Name != defaultMetallbCrName {
+		err := fmt.Errorf("Metallb resource name must be '%s'", defaultMetallbCrName)
+		logger.Error(err, "Invalid Metallb resource name")
```

Comment:
Another thought: now that we are adding status to the metallb crd, would it make sense to update the status of the invalid crd?

Reply:
Will add this once #24 is merged, or add in another PR if this is merged first

Reply:
#24 is merged now, can you add that?